x/crosschain/keeper/keeper_cross_chain_tx.go

```diff
@@ -157,16 +158,39 @@ func (k Keeper) CctxAllPending(c context.Context, req *types.QueryAllCctxPending
 		return nil, status.Error(codes.InvalidArgument, "invalid request")
 	}
 	ctx := sdk.UnwrapSDKContext(c)
+	if req.ChainId > goMath.MaxInt64 {
+		return nil, status.Error(codes.InvalidArgument, "invalid chain id")
+	}
+	pendingCrossChainTxList, err := k.GetActualPendingCCTX(ctx, int64(req.ChainId))
+	if err != nil {
+		return nil, err
+	}
+	return &types.QueryAllCctxPendingResponse{CrossChainTx: pendingCrossChainTxList}, nil
+}
+
+func (k Keeper) CctxAllPendingCount(c context.Context, _ *types.QueryAllCctxPendingCountRequest) (*types.QueryAllCctxPendingCountResponse, error) {
```

Comment:
I think we should have queries in files prefixed with `grpc` to make it consistent throughout the codebase and easier to find